### PR TITLE
update translation table with characters using diacritical marks

### PR DESCRIPTION
**Pull request by carl-textalk**

### CODE REVIEW

**Comment by carl-textalk** [comment]
Hi @bertfrees and @PaulRambags Please review the following change regarding diacritical marks.

**Comment by bertfrees** [comment]
Hi Carl. I don't have a clue about the Swedish braille code, nor have I ever looked into the dotify.translator.impl code. Are you sure you need my review?

**Comment by carl-textalk** [comment]
@bertfrees I see, we just wanted to follow protocol and i understand that the specific Swedish braille code might not be within your area. So, if you don't have any opinions about this repository we will handle this pull request internally.

**Comment by bertfrees** [comment]
OK I think that's best. I'm afraid I don't have anything useful to say.

File: src/org/daisy/dotify/translator/impl/sv_SE/sv_SE-pas2.xml [hunk 1]
```diff
@@ -69,41 +109,70 @@
...
-	<entry key="ü">&#x2833;</entry>
+	<entry key="ü">&#x2833;</entry>										<!-- LATIN SMALL LETTER U WITH DIAERESIS, 00FC-->
+	<entry key="ű">&#x2808;&#x2825;</entry>                    	<!-- LATIN SMALL LETTER U WITH DOUBLE ACUTE , 0171-->
 	<entry key="ù">&#x2808;&#x2825;</entry>
 	<entry key="ú">&#x2808;&#x2825;</entry>
 	<entry key="û">&#x2808;&#x2825;</entry>
 	<entry key="ū">&#x2808;&#x2825;</entry>
+	<entry key="ũ">&#x2808;&#x2825;</entry>						<!-- LATIN SMALL LETTER U WITH TILDE, U+0169 -->
+	<entry key="ǔ">&#x2808;&#x2825;</entry>						<!-- LATIN SMALL LETTER U WITH CARO, U+01D4 -->
+	<entry key="ų">&#x2808;&#x2825;</entry>						<!--LATIN SMALL LETTER U WITH OGONEK, U+0173-->
```
Comment (PaulRambags) [inline src/org/daisy/dotify/translator/impl/sv_SE/sv_SE-pas2.xml]: Please add a space in between the `<!--` and the `LATIN`

File: src/org/daisy/dotify/translator/impl/sv_SE/sv_SE-pas2.xml [hunk 2]
```diff
@@ -69,41 +109,70 @@
...
+	<entry key="ţ">&#x2808;&#x281E;</entry>							<!-- LATIN SMALL LETTER T WITH CEDILLA, 0163 -->
 	<entry key="þ">&#x2808;&#x281E;</entry>
+	<entry key="ț">&#x2808;&#x281E;</entry>							<!-- LATIN SMALL LETTER T WITH COMMA BELOW, U+021B -->
+	<entry key="ť">&#x2808;&#x281E;</entry>							<!-- LATIN SMALL LETTER T WITH CARON, U+0165-->
+	<entry key="ŧ">&#x2808;&#x281E;</entry>							<!-- LATIN SMALL LETTER T WITH STROKE, U+0167-->
+
 	<entry key="u">&#x2825;</entry>
-	<entry key="ü">&#x2833;</entry>
+	<entry key="ü">&#x2833;</entry>										<!-- LATIN SMALL LETTER U WITH DIAERESIS, 00FC-->
+	<entry key="ű">&#x2808;&#x2825;</entry>                    	<!-- LATIN SMALL LETTER U WITH DOUBLE ACUTE , 0171-->
```
Comment (PaulRambags) [inline src/org/daisy/dotify/translator/impl/sv_SE/sv_SE-pas2.xml]: The above two lines don't align with the rest of the file

File: src/org/daisy/dotify/translator/impl/sv_SE/sv_SE-pas2.xml [hunk 3]
```diff
@@ -41,26 +51,56 @@
...
+	<entry key="ļ">&#x2808;&#x2807;</entry>							<!-- LATIN SMALL LETTER L WITH CEDILLA, U+013C -->
+	<entry key="ľ">&#x2808;&#x2807;</entry>							<!-- LATIN SMALL LETTER L WITH CARON, U+013E -->
+
 	<entry key="m">&#x280D;</entry>
+
 	<entry key="n">&#x281D;</entry>
 	<entry key="ñ">&#x2808;&#x281D;</entry>
 	<entry key="ń">&#x2808;&#x281D;</entry>						<!-- LATIN SMALL LETTER N WITH ACUTE, 0144 -->
+	<entry key="ņ">&#x2808;&#x281D;</entry>						<!-- LATIN SMALL LETTER N WITH CEDILLA, U+0146 -->
+	<entry key="ň">&#x2808;&#x281D;</entry>						<!--  LATIN SMALL LETTER N WITH CARON, U+0148 -->
```
Comment (PaulRambags) [inline src/org/daisy/dotify/translator/impl/sv_SE/sv_SE-pas2.xml]: Please remove a space in between the `<!--` and the `LATIN`

**Review by PaulRambags — APPROVED** [review]
I have added some remarks on white space. Apart from that, I don't know how the Swedish braille tables should be, so I approve this PR.

**Review by kalaspuffar — APPROVED** [review]

**Comment by kalaspuffar** [comment]
Ran the regression tests and found only expected changes.

**Pull request merged**